.github/workflows/unit.yml

```diff
@@ -20,29 +20,38 @@ jobs:
       GO111MODULE: "on"
 
     steps:
-      - name: Setup Go ${{ matrix.go-version }}
+      - if: ${{ matrix.go-version == '1.14' }}
```

Comment:
It feels like this step shouldn't be a conditional, we want to always install the latest 1.x go.